modules/ossm-config-openshift-monitoring-kiali.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/observability/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-kiali_{context}"]
+= Configuring OpenShift Monitoring with Kiali
+:context: ossm-config-openshift-monitoring-kiali
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. Content came from different places, so the first priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {KialiProduct} with `user-workload monitoring`.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* `user-workload monitoring` is enabled. See "Enabling user-defined projects".
+
+* {KialiProduct} 1.86 is installed.
+
+.Procedure
+
+. Create a `ClusterRoleBinding` resource for Kiali:
++
+.Example `ClusterRoleBinding` configuration
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kiali-monitoring-rbac
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view
+subjects:
+- kind: ServiceAccount
+  name: kiali-service-account
+  namespace: istio-system
+----
+
+. Create a Kiali resource and point it to your Istio instance:
++
+.Example Kiali resource configuration
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali-user-workload-monitoring
+  namespace: istio-system
+spec:
+  external_services:
+    prometheus:
+      auth:
+        type: bearer
+        use_kiali_token: true
+      thanos_proxy:
+        enabled: true
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
+----
+
+. Get the Kiali URL from the `Route`:
++
+[source,terminal]
+----
+echo "https://$(kubectl get routes -n istio-system kiali -o jsonpath='{.spec.host}')"
```

Comment:
Sorry this should probably be `oc` for openshift:
```suggestion
echo "https://$(oc get routes -n istio-system kiali -o jsonpath='{.spec.host}')"
```

Reply:
Users also need to wait for the Kiali resource to be ready after creating it in the previous step in order for this command to work. Probably need a note indicating that or else I can provide an `oc wait ...` command to run until this will work?

Reply:
OK. I revised the next step to say: "When the Kiali resource is ready, get the Kiali URL from the `Route` using the following command:"